### ISSUE TITLE
Feature request: Release an Android app

### ISSUE THREAD

**Issue by expertmanofficial** [Dimillian/IceCubesApp#1551]
I am on Android and I don't own , don't have an iPhone.

But I wish I could try your app on Android - could you do it please?

Thanks in advance.

**Comment by Havhingstor** [comment]
I don't think this is going to be done anytime soon (at least by @Dimillian) since the app is mainly SwiftUI, which doesn't exist on Android, so a release on Android would need an extensive / complete rewrite. Additionally, this would require somewhat extensive knowledge of a similar Android framework, which at least I don't have (can't speak for other contributors though)

**Comment by expertmanofficial** [comment]
Still need this to be released.
